packages/app/cypress/e2e/integration/top-nav.spec.ts

```diff
@@ -289,31 +289,118 @@ describe('App Top Nav Workflows', () => {
           name: 'Profile Settings',
           href: 'https://on.cypress.io/dashboard/profile',
         })
+      })
+
+      it('replaces user avatar after logout', () => {
+        cy.get('@logInButton').click()
+
+        cy.withCtx((ctx) => {
+          sinon.stub(ctx._apis.authApi, 'logOut').callsFake(async () => {
+            // resolves
+          })
+        })
 
         cy.intercept('mutation-Logout').as('logout')
 
         cy.findByRole('button', { name: 'Log Out' }).should('be.visible').click()
```

Comment:
Side note, we get the visibility check for free before `.click()` so we don't strictly need it here.